templates/terraform/encoders/compute_region_disk_resource_policies_attachment.go.erb

```diff
@@ -0,0 +1,17 @@
+config := meta.(*Config)
+project, err := getProject(d, config)
+if err != nil {
+  return nil, err
+}
+
+region, err := getRegion(d, config)
+if err != nil {
+	return nil, err
+}
+if region == "" {
+	return nil, fmt.Errorf("zone must be non-empty - set in resource or at provider-level")
```

Comment:
drive-by: s/zone/region?

Reply:
Whoops, missed that one.

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/pull/3229